standard/ERCs/erc-6900.md

```diff
@@ -425,11 +435,11 @@ struct PluginManifest {
 
 #### Responsibilties of `StandardExecutor` and `PluginExecutor`
 
-`StandardExecutor` functions are mainly used for open-ended execution of external contracts.
+`StandardExecutor` functions are mainly used for open-ended execution against external addresses.
```

Comment:
Hmm yeah. I just thought "execution of an address" was a weird way to put things, since EOA's don't have any functions you can execute.

perhaps "open-ended calls of external addresses"?

Reply:
Yeah that sounds goo,  "`StandardExecutor` functions are used for open-ended calls to external addresses"